ajax/libs/slim-select/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "slim-select",
+  "description": "Slim vanilla javascript select dropdown",
+  "author": "Brian Voelker <brian@webiswhatido.com> (http://webiswhatido.com)",
+  "homepage": "https://slimselectjs.com",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brianvoe/slim-select.git"
+  },
+  "keywords": [
+    "select",
+    "vanilla",
+    "dropdown",
+    "search"
+  ],
+  "npmName": "slim-select",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "index.js",
+        "index.css"
+      ]
+    }
+  ],
+  "filename": "index.js"
```

Comment:
Let's move `"filename": "index.js"` above.
Let it below the `"description": "Slim vanilla javascript select dropdown",` is better.
It will let people find what file is the main file of library at first glance.